unfoldingWord/translationCore
# Fix Dialog Message For Local User Upload

#### This pull request addresses:
This PR fixes the dialog message that appears when a local user attempts to upload a project.

#### How to test this pull request:
Login as a local user and try to upload a project to door43 you should be prompted a error message stating that action is not possible


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2499)
